format/spec.md

```diff
@@ -702,6 +703,41 @@ Blob metadata is a struct with the following fields:
 | _optional_ | _optional_ | **`properties`** | `map<string, string>` | Additional properties associated with the statistic. Subset of Blob properties in the Puffin file. |
 
 
+#### Partition statistics
+
+Partition statistics files are based on [Partition Statistics file spec](#partition-statistics-file). Partition statistics are informational. A reader can choose to
```

Comment:
I wonder if we can simplify this a bit. I think we end up being a bit repetitive here. I'm wondering if we can just get one sentence per point here.

1. Partition statistics are not required for reading or planning and readers may ignore them.
2. There is at most one partition statistics file per snapshot
3. Any snapshot listed in the `snapshots` of the metadata may have a partition statistics file
4. (?) Partition statistics may not be associated with a snapshot that doesn't exist in the metadata. // Do we want to make this a requirement?

Then we have some optional implementation guidance,

1. Partitions statistics can be added to the metadata during write operations

That's just my feedback, I just am leaning towards a more straightforward description here.

Reply:
Simplified